### PR TITLE
LPS-180324 Add test to get name field of flattened categories of a vocabulary

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-180324

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Tim-Cao** [comment]
ci:test:sf

File: modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/TaxonomyCategoryResourceTest.java [hunk 1]
```diff
@@ -310,6 +324,29 @@ private TaxonomyCategory _addTaxonomyCategoryWithParentTaxonomyCategory(
 			parentTaxonomyCategoryId, taxonomyCategory);
 	}
 
+	private JSONObject _invoke(
```
Comment (jkappler) [inline modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/TaxonomyCategoryResourceTest.java:327]: Hi @Tim-Cao, I think we can avoid invoking like we did with the other test. I'll check it out on Monday and ping you on this.

Thanks!

Comment (Tim-Cao) [reply modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/TaxonomyCategoryResourceTest.java:327]: Thanks in advance! I didn't find the correct way to pass fields parameter.

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**